emr_run_spark.py

```diff
@@ -245,7 +245,7 @@ def _get_step_ids_for_job_flow(job_flow_id, client):
 
 def _wait_for_job_flow(aws_region, job_flow_id, step_ids=[]):
     while True:
-        time.sleep(5)
+        time.sleep(30)
```

Comment:
I change it long time ago cause I noticed that Amazon throttled us in 5 seconds and it was not necessary to sample them every 5 seconds.